homeassistant/components/advantage_air/switch.py

```diff
@@ -62,6 +65,32 @@ async def async_turn_off(self, **kwargs: Any) -> None:
         await self.async_update_ac({"freshAirStatus": ADVANTAGE_AIR_STATE_OFF})
 
 
+class AdvantageAirMyFan(AdvantageAirAcEntity, SwitchEntity):
+    """Representation of Advantage Air MyFan control."""
+
+    _attr_icon = "mdi:fan-auto"
+    _attr_name = "MyFan"
```

Comment:
Use translations https://developers.home-assistant.io/docs/internationalization/core#entities

Reply:
Is this how it's called by the vendor as `MyFan` seems a bit strange as name for an auto mode?

Reply:
Yes, the vendor loves their "My" names: https://www.advantageair.com.au/wp-content/uploads/2019/10/MyComfort.pdf

This is technically a branded feature which only exists in English, so I am not sure translations help much, but I'll add it in as requested.

Reply:
Right. Skip the translation key as otherwise they will translate it to same anyway. Unnecessary work for people.